yamllint/config.py

```diff
@@ -153,8 +153,29 @@ def validate_rule_conf(rule, conf):
         return False
 
     if isinstance(conf, dict):
-        if ('ignore' in conf and
-                not isinstance(conf['ignore'], pathspec.pathspec.PathSpec)):
+        if 'ignore-from-file' in conf and isinstance(
+            conf['ignore-from-file'], (str, list)
+        ):
```

Comment:
Is `isinstance(conf['ignore-from-file'], (str, list))` useful? If yes, I don't understand, can you explain? (I would have copied the same logic as for `ignore`, a few lines below.)

It seems to me that it would prevent displaying a nice helpful error `invalid config: ...` if the user made a mistake like `ignore-from-file: 42`.

Reply:
> Is isinstance(conf['ignore-from-file'], (str, list)) useful? If yes, I don't understand, can you explain? (I would have copied the same logic as for ignore, a few lines below.)

I was only handling those two cases, so it seemed apt. However, I changed it to follow the same check as `ignore`'s.